packages/user/src/components/UsersTable/UserActions.tsx

```diff
@@ -0,0 +1,65 @@
+import { useTranslation } from "@dzangolab/react-i18n";
+import { ActionsMenu, ConfirmationModal } from "@dzangolab/react-ui";
+import { MenuItem } from "primereact/menuitem";
+import { useState } from "react";
+
+export const UserAction = () => {
+  const { t } = useTranslation("users");
+  const [showEnableConfirmation, setShowEnableConfirmation] = useState(false);
+  const [showDisableConfirmation, setShowDisableConfirmation] = useState(false);
+
+  const actionItems: MenuItem[] = [
+    {
+      label: t("table.actions.enableUser"),
+      icon: "pi pi-check",
+      command: () => {
+        setShowEnableConfirmation(true);
+      },
+    },
+    {
+      label: t("table.actions.disableUser"),
+      icon: "pi pi-times",
+      command: () => {
+        setShowDisableConfirmation(true);
+      },
+    },
```

Comment:
This action should be displayed in red. We have something for this already, check things or invitations table.